mmv1/third_party/terraform/services/datalossprevention/resource_data_loss_prevention_discovery_config_test.go

```diff
@@ -1292,3 +1365,249 @@ resource "google_data_loss_prevention_discovery_config" "basic" {
 }
 `, context)
 }
+
+func testAccDataLossPreventionDiscoveryConfig_dlpDiscoveryConfigStartAwsS3(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "project" {
+}
+resource "google_organization_iam_member" "service_agent" {
```

Comment:
These IAM grants are the reason that the tests are failing on our CI system - the service account running the test doesn't have permission to make org-level IAM changes. What are the minimum IAM roles needed to create the config?

My reading of [the docs](https://cloud.google.com/sensitive-data-protection/docs/data-profiles#user-roles-org-folder) makes me think that granting the SA running the test `roles/dlp.admin` at the org level will be enough to run the test:

>If you don't have the Organization Administrator (roles/resourcemanager.organizationAdmin) or Security Admin (roles/iam.securityAdmin) role, you can still create a scan configuration. After you create the scan configuration, someone in your organization who has one of these roles must grant discovery access to the service agent.